air/src/passes/translate_from_mir.rs

```diff
@@ -604,6 +570,79 @@ impl AirBuilder<'_> {
     fn insert_op(&mut self, op: Operation) -> NodeIndex {
         self.air.constraint_graph_mut().insert_node(op)
     }
+
+    /// Extracts the trace access information from a given [Mir] `Boundary``.
+    fn extract_trace_from_boundary(
+        &self,
+        boundary: MirBoundary,
+    ) -> Result<MirTraceAccess, CompileError> {
```

Comment:
nit: there is an extra backtick at the end of line 574.

Also, could we expand the doc comment a bit more to explain what this does? My current understanding is that given a boundary constraint expression, this figures out which trace column this boundary constraint references - correct?